scala/private/rules/scala_library.bzl

```diff
@@ -135,8 +135,11 @@ def _lib(
 ##
 
 def _scala_library_impl(ctx):
-    if ctx.attr.jvm_flags:
-        print("'jvm_flags' for scala_library is deprecated. It does nothing today and will be removed from scala_library to avoid confusion.")
+    if hasattr(ctx.attr, "jvm_flags"):
```

Comment:
do we need this? Won't bazel fail if users pass attributes that the rule doesn't support?

Reply:
Ah, right you are! Didn't think about that.